tests/aws/services/apigateway/test_apigateway_basic.py

```diff
@@ -157,7 +157,8 @@ def test_delete_rest_api_with_invalid_id(self, aws_client):
         assert "foobar" in e.value.response["Error"]["Message"]
 
     @pytest.mark.parametrize("url_function", [path_based_url, host_based_url])
-    @markers.aws.unknown
+    @markers.aws.needs_fixing
+    # This is not a possible feature on aws. Is there a reason for us to add that support?
```

Comment:
This is a specific feature so that people can always have the same API ID when recreating their state. I think it avoids the whole custom domain. I think it has some use case, if you're developing something quickly and restart LocalStack to redeploy some code, you don't need to redeploy your own code with the new API ID as well. 
```suggestion
    @markers.aws.only_localstack
    # This is not a possible feature on aws.
```

Reply:
oops! I had a last commit on my machine changing this.